course3/homework-string/my_string.cc

```diff
@@ -0,0 +1,192 @@
+#include "my_string.h"
+
+#include <cstring>
+#include <memory>
+
+MyString::MyString() 
+	:
+	mBuffer(nullptr)
```

Comment:
this is not necessary, as any unique_ptr is null by default (initialized to null in it's own default constructor).
This means that the entire constructor is unnecessary.